sensor_msgs/PointCloud2 cloud_in
bool homing
string marker_file
---
sensor_msgs/PointCloud2 cloud_out
